### ISSUE TITLE
node-red latest docker image run with dmesg error: [  703.155850] node (3076): undefined instruction: pc=64bd2338

### ISSUE THREAD

**Issue by SuiKaSan** [node-red/node-red#4665]
### Current Behavior

Running node-red docker image in my server and got the error repeatly.

[  703.155850] node (3076): undefined instruction: pc=64bd2338
[  703.155890] CPU: 1 PID: 3076 Comm: node Tainted: G           O      4.19.111 #46
[  703.155897] Hardware name: Generic DT based system
[  703.155907] PC is at 0x17fdb5e
[  703.155913] LR is at 0xa486e7
[  703.155928] pc : [<017fdb5e>]    lr : [<00a486e7>]    psr: 00000030
[  703.155934] sp : aee89af8  ip : aee89af8  fp : 00000000
[  703.155943] r10: 0430a480  r9 : a6f233cc  r8 : 00000001
[  703.155953] r7 : aee89af8  r6 : 04381df8  r5 : 00000000  r4 : 04381c68
[  703.155962] r3 : 00000008  r2 : 04381d6c  r1 : 00000000  r0 : 00000000
[  703.155972] Flags: nzcv  IRQs on  FIQs on  Mode USER_32  ISA Thumb  Segment user
[  703.155982] Control: 10c5387d  Table: 35c7806a  DAC: 00000055
[  703.155996] Code: 4770 ff00 0c40 4770 (efa0) 0e00
[  703.156066] node (3076): undefined instruction: pc=d0f67d8f
[  703.156080] CPU: 1 PID: 3076 Comm: node Tainted: G           O      4.19.111 #46
[  703.156091] Hardware name: Generic DT based system
[  703.156098] PC is at 0x17fdb4c
[  703.156106] LR is at 0xa48713
[  703.156115] pc : [<017fdb4c>]    lr : [<00a48713>]    psr: 00000030
[  703.156123] sp : aee89af8  ip : aee89af8  fp : 00000000
[  703.156130] r10: 0430a480  r9 : a6f233cc  r8 : 00000001
[  703.156136] r7 : aee89af8  r6 : 04381df8  r5 : 00000000  r4 : 04381c68
[  703.156143] r3 : 00000008  r2 : 04381d6c  r1 : 00000000  r0 : 00000000
[  703.156152] Flags: nzcv  IRQs on  FIQs on  Mode USER_32  ISA Thumb  Segment user
[  703.156161] Control: 10c5387d  Table: 35c7806a  DAC: 00000055
[  703.156173] Code: 4770 ec51 0f1e 4770 (ffb0) 0300
[  703.156192] node (3076): undefined instruction: pc=1c92085a
[  703.156204] CPU: 1 PID: 3076 Comm: node Tainted: G           O      4.19.111 #46
[  703.156212] Hardware name: Generic DT based system
[  703.156220] PC is at 0x17fdb52
[  703.156230] LR is at 0xa486fd
[  703.156239] pc : [<017fdb52>]    lr : [<00a486fd>]    psr: 00000030
[  703.156248] sp : aee89af8  ip : aee89af8  fp : 00000000
[  703.156256] r10: 0430a480  r9 : a6f233cc  r8 : 00000001
[  703.156266] r7 : aee89af8  r6 : 04381df8  r5 : 00000000  r4 : 04381c68
[  703.156274] r3 : 00000008  r2 : 04381d6c  r1 : 00000000  r0 : 00000000
[  703.156283] Flags: nzcv  IRQs on  FIQs on  Mode USER_32  ISA Thumb  Segment user
[  703.156292] Control: 10c5387d  Table: 35c7806a  DAC: 00000055
[  703.156304] Code: 4770 ffb0 0300 4770 (ef00) 0c40
[  703.156322] node (3076): undefined instruction: pc=aa994fb1
[  703.156334] CPU: 1 PID: 3076 Comm: node Tainted: G           O      4.19.111 #46
[  703.156342] Hardware name: Generic DT based system
[  703.156351] PC is at 0x17fdb58
[  703.156360] LR is at 0xa486d1
[  703.156366] pc : [<017fdb58>]    lr : [<00a486d1>]    psr: 60000030
[  703.156372] sp : aee89af8  ip : aee89af8  fp : 00000000
[  703.156378] r10: 0430a480  r9 : a6f233cc  r8 : 00000001
[  703.156389] r7 : aee89af8  r6 : 04381df8  r5 : 00000000  r4 : 04381c68
[  703.156397] r3 : 00000008  r2 : 04381d6c  r1 : 00000000  r0 : 00000000
[  703.156406] Flags: nZCv  IRQs on  FIQs on  Mode USER_32  ISA Thumb  Segment user
[  703.156416] Control: 10c5387d  Table: 35c7806a  DAC: 00000055
[  703.156426] Code: 4770 ef00 0c40 4770 (ff00) 0c40
[  706.229804] bt_err_ratelimited: 13 callbacks suppressed

![lQLPJx7BWB0V6ZV6zQWwsHb9SiHw6Wk7BhLAPfKXnQA_1456_122](https://github.com/node-red/node-red/assets/44567811/b8b0a9f4-5fa0-4fed-a532-92090dbc414b)

### Expected Behavior

No error.

### Example flow

```
paste your flow here
```

### Environment

- Node-RED version: latest
- Node.js version: 16
- npm version: none
- Platform/OS: docker
- Browser: none

**Comment by SuiKaSan** [comment]
The error cause the server to acted unstale.
What's wrong with it?

**Comment by hardillb** [comment]
This is unlikely to be a Node-RED issue.

What hardware are you running this on? Best guess this is an ARM machine that doesn't support one of the instructions that the NodeJS JIT compiler is generating.

**Comment by S474N** [comment]
Same problem, tested upgrading, downgrading,. Nothing :(

Last Docker version, on Rpi, newest system.

**Comment by hardillb** [comment]
Another option is that it's a node with native code

Please also supply a list of installed nodes

**Comment by brybad** [comment]
I had the same issue with the latest-debian image. Switching to 3.1.9-debian worked perfectly.

Docker Desktop on Windows 10...

**Comment by hardillb** [comment]
`latest-debian` and `3.1.9-debian` are currently the same image

**Comment by brybad** [comment]
Yes, they are the same but they weren't being pulled in the same way for some reason and no matter how many times I tried latest-debian, it would never run until I used 3.1.9-debian.
The alpine versions ran with no issues and why I mentioned the debian version.

**Comment by knolleary** [comment]
Closing as part of a tidy-up of the issue tracker — this has been inactive for a long time and much has changed in Node-RED since. If it's still relevant on a current version, please open a fresh issue with up-to-date details and we'll take a look. Thanks for raising it.
